doc/user/bgp.rst

```diff
@@ -2004,31 +2014,58 @@ Displaying BGP Information
 
          Total number of prefixes 1
 
-.. index:: show ip bgp regexp LINE
-.. clicmd:: show ip bgp regexp LINE
+Some other commands are available on that format, and permit to dump either IPv4
+routing table or IPv6 routing table, depending on the presence or not of the
+`ip` keyword.
+
+.. index:: show [ip] bgp regexp LINE
+.. clicmd:: show [ip] bgp regexp LINE
 
    This command displays BGP routes using AS path regular expression
    (:ref:`bgp-regular-expressions`).
 
-.. index:: show bgp <ipv4|ipv6> summary
-.. clicmd:: show bgp <ipv4|ipv6> summary
+.. index:: show [ip] bgp summary
+.. clicmd:: show [ip] bgp summary
 
    Show a bgp peer summary for the specified address family.
 
-.. index:: show bgp <ipv4|ipv6> neighbor [PEER]
-.. clicmd:: show bgp <ipv4|ipv6> neighbor [PEER]
+The old command structure :cliref:`show ip bgp` may become deprecated in the
```

Comment:
how about s/may/is ?

Reply:
actually deprecated, that would mean that we use DEFUN_DEPRECATED or CPP_NOTICE to inform developers that the command will not be supported.
So I think this makes sense to add may be removed.